sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsAsyncClientTests.java

```diff
@@ -41,234 +38,371 @@ protected void afterTest() {
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
     public void createRoomFullCycleWithResponseStep(HttpClient httpClient) {
-        roomsAsyncClient = setupAsyncClient(httpClient, "createRoomFullCycleWithResponse");
+        roomsAsyncClient = setupAsyncClient(httpClient,
+                "createRoomFullCycleWithResponse");
         assertNotNull(roomsAsyncClient);
-        Mono<Response<CommunicationRoom>> response1 = roomsAsyncClient.createRoomWithResponse(VALID_FROM, VALID_UNTIL, roomJoinPolicy, participants1);
+
+        CreateRoomOptions roomOptions = new CreateRoomOptions()
+                .setValidFrom(VALID_FROM)
+                .setValidUntil(VALID_UNTIL)
+                .setParticipants(participants1);
+
+        Mono<Response<CommunicationRoom>> response1 = roomsAsyncClient.createRoomWithResponse(roomOptions);
 
         List<String> roomParticipantsMri = new ArrayList<String>();
```

Comment:
Where is roomParticipantsMri being used? Clean up if not used

Reply:
Yeah, I think I added that to test something, removed now.